src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -779,7 +779,45 @@ public IEnumerable<Element> GetJoinedElements()
                 .ToList();
         }
 
+        /// <summary>
+        /// Unjoins elements if they are joined 
+        /// </summary>
+        /// <param name="elements">List of elements to unjoin</param>
+        /// <returns>Elements that have been unjoined</returns>
+        public static List<Element> UnjoinGeometry(List<Element> elements)
+        {
+            List<Element> modifiedElements = new List<Element>();
+            for (int i = 0; i < elements.Count; i++)
+            {
+                List<Element> joinedElements = JoinGeometryUtils.GetJoinedElements(Document, elements[i].InternalElement)
+                                                                .Select(id => Document.GetElement(id).ToDSType(true))
+                                                                .ToList();
+                if (joinedElements.Count > 0)
```

Comment:
We should bail early instead of wrapping everything in an `if`
```suggestion
                if (joinedElements.Count <= 0) return modifiedElements;
```

Reply:
If i `return modifiedElements` here would it go through the rest of the elements as it needs to?

made a `continue` for now.

Reply:
if it needs to continue looping then `continue` is the right choice 👍